_content/essays/iridescence-on-the-water_geoff.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "An Iridescence on the Water : The Teachings of Chao Khun Nararatana Rajamanit"
+authors:
+  - geoff
+external_url: "https://www.dhammatalks.org/Archive/Writings/ThaiAjaans/ChaoKhunNararatana.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1ITgASboDbSRVk-zvuxzAJ3pFu1AMntBZ/view?usp=sharing"
+course: thai
+tags:
+  - path
+year: 1997
+pages: 6
+---
+
+> We must practice putting the mind back into shape. Before we do anything,
+while we’re doing it, and after it’s done, we have to practice keeping the mind
+cheerful and bright, with a constant sense of well-being.
+
+A concise overview of the teachings of Chao Khun Nararatana Rajamanit, a respected yet lesser-known teacher of the Thai Forest tradition.
```

Comment:
He was not in the forest tradition... He was firmly in the Thai city monk tradition.